homeassistant/components/esphome/light.py

```diff
@@ -391,3 +412,13 @@ def min_mireds(self) -> int:
     def max_mireds(self) -> int:
```

Comment:
Drop the `min_mireds` and `max_mireds` as well since they are deprecated
https://github.com/home-assistant/core/blob/053eaad2bd09ac1b656be5ae22b379f36891a295/homeassistant/components/light/__init__.py#L206

Reply:
I disagree with this suggestion; I would argue that removing a deprecated property should be its own PR. The presence of a deprecated property is not related to the bug in question, and changing this might break some setups. Also, there are several other light integrations that still use the min/max mired property, so I see no reason to remove the property in a bugfix PR.

Reply:
I don't agree so I marked this as `second-opinion-wanted` to give another member a chance to weigh in.